### DISCUSSION TITLE
Can't find domain after transfer old personal account to new personal account

### DISCUSSION THREAD

**Discussion by kbchs** [vercel/community#4647] [Help]
### Summary

I created a new account, which by default has a team account [(Default teams apply only to accounts created after this feature was enabled in September 2023)](https://vercel.com/docs/accounts/create-a-team#default-team). Then, while moving a domain from my old personal account to the new team account, I mistakenly moved it to the new personal account. Now I can't find the domain in the new team account. How can I find the domain or move the domain from a new personal account to a new team account?

### It's a new team account:
![Screenshot 2023-10-28 110802](https://github.com/vercel/community/assets/149218582/bba480e4-3d95-4dba-8e4a-62a31a4b2c3a)

### It's a new personal account:
![Screenshot 2023-10-28 110844](https://github.com/vercel/community/assets/149218582/87d7da8f-ec27-4f52-bd73-d51bb146e361)

**Comment by amyegan** [comment]
Sorry you ran into this problem. If you let me know the domain, I can look into it for you! 🙏

**Reply by kbchs** [reply]
The domain is kbchs.edu.bd. Please help me to find out the domain & make sure it'll be in KBCHS Team account as soon as possible.

**Reply by amyegan** [reply]
From what I'm able to see, it looks like that domain is already owned by your team and assigned to your project. I reached out to the engineering team for help investigating why it doesn't appear on your Domains page. I'll post updates here when more info is available.

**Reply by kbchs** [reply]
Thank you. Please figure out the issue. It’s a important domain for us. We will be loss many things, if we can't control it or manage it. Thanks a lot for your time.
